rhel8/profiles/cis_level2_server.profile

```diff
@@ -8,12 +8,14 @@ metadata:
 
```

Comment:
I just wanted to point out that renaming this file means renaming a profile ID and this breaks compatibility with previous versions and tailoring, in other words if people has automated scanning that uses old 'cis' profile, it will break because the profile won't exist anymore in newer versions. We should probably keep the way it is and may add `cis_level2_server.profile` that will extend the `cis` profile, but this is also not ideal.

Reply:
That makes sense - thanks for taking a look :smile: 

I was taking some inspiration from the Ubuntu PR at #6416, but after the discussion going on in #6996, I think the conclusion might be that we wait for tags to be available in `.policy` files and then I can rework this PR to just make edits to the original `cis.profile` instead of splitting it out.

Reply:
I agree with @ggbecker . Currently, the profile contains probably mix of server and workstation rules, both level 1 and level 2. We have to keep this profile ID and I would suggest to use "cis" ID for "Level 2 server".

Reply:
I agree. I think if we have to maintain the original ID to avoid breaking people's scripts, "Level 2 - Server" makes sense to be the one to pick. If they're using the current profile then they're already applying or auditing against the Level 2 controls so this approach avoids unexpectedly switching people to a more lenient profile if we started using "cis" to refer to Level 1. We can use other IDs for the other 3 profiles.